code/obj/machinery/phosphorizer.dm

```diff
@@ -0,0 +1,163 @@
+/obj/machinery/phosphorizer
+	name = "phosphorizer"
+	desc = "A device capable of baking a phosphor onto light tubes or bulbs, changing the color of light they emit."
+	icon = 'icons/obj/machines/phosphorizer.dmi'
+	icon_state = "baseunit"
+	anchored = 1
+	density = 1
+	power_usage = 150
```

Comment:
```suggestion
	power_usage = 150
	status = REQ_PHYSICAL_ACCESS
```
Given that you've gone for a "physical knobs" style interface, I think making it require physical access seems right.

Reply:
Seems sensible. f7358dc